src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -83,10 +84,25 @@ public User findUserByEmail(String email) throws NotFoundException {
         return user;
     }
 
+    @Deprecated //instead use getUsersByUserRole(user, roleName)
     public Collection<User> describeUsers() {
         return userDAO.findUsers();
     }
 
+    public List<User> getUsersByUserRole(User user, String roleName) {
+        if (roleName.equalsIgnoreCase(UserRoles.SIGNINGOFFICIAL.getRoleName())) {
+            if (Objects.nonNull(user.getInstitutionId())) {
+                return userDAO.findUsersByInstitution(user.getInstitutionId());
+            } else {
+                throw new NotFoundException("Signing Official (user: " + user.getDisplayName() + ") "+ "is not associated with an Institution.");
```

Comment:
nit, unnecessary string concats:
```suggestion
                throw new NotFoundException("Signing Official (user: " + user.getDisplayName() + ") is not associated with an Institution.");
```